packages/theme-chalk/src/form.scss

```diff
@@ -222,9 +222,38 @@ $form-item-label-top-margin-bottom: map.merge(
         box-shadow: 0 0 0 1px getCssVar('color-danger') inset;
       }
     }
+
     .#{$namespace}-input__wrapper {
-      box-shadow: 0 0 0 1px getCssVar('color-danger') inset;
+      box-shadow: 0 0 0 1px getCssVar('color-danger') inset ;
+      &:hover {
+        box-shadow: 0 0 0 1px getCssVar('color-danger') inset
+      }
+      &.is-focus {
+        box-shadow: 0 0 0 1px getCssVar('color-danger') inset !important;
+      }
+    }
+
+    .#{$namespace}-select {
+       &:hover{
+         box-shadow: 0 0 0 1px transparent;
+       }
+
+      .#{$namespace}-input{
+        .#{$namespace}-input__wrapper {
+          &:hover{
```

Comment:
## Suggest

I would appreciate it if you could organize the code convention.

```
// Braces
.#{$namespace}-input {

}

// Semicolon
box-shadow: 0 0 0 1px getCssVar('color-danger') inset;
```

1. Semicolon
2. Braces